pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -62,6 +62,10 @@ func fixBundleCached() error {
 
 // Check if oc binary is cached or not
 func checkOcBinaryCached() error {
+	// Remove oc binary from older location and ignore the error
+	// We should remove this code after 3-4 releases.
+	os.Remove(filepath.Join(constants.CrcBinDir, "oc"))
```

Comment:
I think gosec will complain if it's not `_ = os.Remove(...)`

Reply:
@cfergeau it does put the severity is not hight so CI ignore it like any other errors like we don't handle the error for `fmt.Printf`.